native/packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBar.android.js

```diff
@@ -97,6 +98,7 @@ export type Props = {
   leadingButton: ?Element<BpkNavigationBarButtonAndroid>,
   trailingButton: ?Element<BpkNavigationBarButtonAndroid>,
   subtitleView: ?Element<any>,
+  style: any,
```

Comment:
Actually, sorry the type here should be `?(Object | Array<Object>)`

like 
https://github.com/Skyscanner/backpack/blob/c70f45082ff58d0d8971dd3d451fdc151bf44a18/native/packages/react-native-bpk-component-button-link/src/common-types.js#L39
https://github.com/Skyscanner/backpack/blob/3893710c9d7ba1957bf71bc7343a4bf1a79c8a67/native/packages/react-native-bpk-component-phone-input/src/BpkFlag.js#L31

Reply:
Didn't we have issues with that specifically? cc @matthewdavidson